pkg/bench/rttanalysis: BenchmarkJobs failed [unexpectedly no result routers during IMPORT job]

pkg/bench/rttanalysis.BenchmarkJobs [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachprodMicrobenchmarkWeeklyWip/20289400?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachprodMicrobenchmarkWeeklyWip/20289400?buildTab=artifacts#./artifacts/microbench/experiment) on master @ [ca0d4076c381609d6487040141c87e36a68b211f](https://github.com/cockroachdb/cockroach/commits/ca0d4076c381609d6487040141c87e36a68b211f):

Fatal error:

```
panic: unexpectedly no result routers in {0xc01c3953b0 [] [] {[]} 0 local}

```
Stack: 

```
goroutine 403358 gp=0xc0035acf00 m=10 mp=0xc000980008 [running]:
panic({0x5525780?, 0xc02090f050?})
	GOROOT/src/runtime/panic.go:811 +0x168 fp=0xc009b5e320 sp=0xc009b5e270 pc=0x4b24c8
github.com/cockroachdb/cockroach/pkg/sql/physicalplan.(*PhysicalPlan).GetResultTypes(0x4b093f?)
	pkg/sql/physicalplan/physical_plan.go:221 +0xb3 fp=0xc009b5e370 sp=0xc009b5e320 pc=0x29702b3
github.com/cockroachdb/cockroach/pkg/sql/physicalplan.(*PhysicalPlan).EnsureSingleStreamOnGateway(0xc00dcc2380, {0x66707a0, 0xc01fcae5a0}, 0x0)
	pkg/sql/physicalplan/physical_plan.go:502 +0x78 fp=0xc009b5e558 sp=0xc009b5e370 pc=0x2971d98
github.com/cockroachdb/cockroach/pkg/sql.finalizePlanWithRowCount({0x66707a0, 0xc01fcae5a0}, 0x0?, 0xc00dcc2380, 0x0?)
	pkg/sql/distsql_physical_planner.go:5575 +0x66 fp=0xc009b5e7e0 sp=0xc009b5e558 pc=0x3ed71c6
github.com/cockroachdb/cockroach/pkg/sql.FinalizePlan(...)
	pkg/sql/distsql_physical_planner.go:5562
github.com/cockroachdb/cockroach/pkg/sql/importer.distImport.func1(...)
	pkg/sql/importer/import_processor_planning.go:114
github.com/cockroachdb/cockroach/pkg/sql/importer.distImport({_, _}, {_, _}, _, _, {_, _, _}, {0x0, ...}, ...)
	pkg/sql/importer/import_processor_planning.go:118 +0x1719 fp=0xc009b5ebb0 sp=0xc009b5e7e0 pc=0x47a96b9
github.com/cockroachdb/cockroach/pkg/sql/importer.ingestWithRetry({_, _}, {_, _}, _, _, {_, _, _}, {0x0, ...}, ...)
	pkg/sql/importer/import_job.go:675 +0xa2f fp=0xc009b5efc8 sp=0xc009b5ebb0 pc=0x479a56f
github.com/cockroachdb/cockroach/pkg/sql/importer.(*importResumer).Resume(0xc00d00b040, {0x666ef58, 0xc03654d8f0}, {0x57313a0?, 0xc023403660})
	pkg/sql/importer/import_job.go:238 +0x9d5 fp=0xc009b5f550 sp=0xc009b5efc8 pc=0x4796a75
github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine.func2(0xc036556d80?, 0x0?, 0xc009b5f928, {0x666a0c0?, 0xc00d00b040?}, {0x666ef58?, 0xc03654d8f0?}, {0x57313a0?, 0xc023403660?})
	pkg/jobs/registry.go:1607 +0xb2 fp=0xc009b5f5b0 sp=0xc009b5f550 pc=0x26360d2
github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine(0xc025509508, {0x66707a0, 0xc002f1b400}, {0x57313a0, 0xc023403660}, {0x666a0c0, 0xc00d00b040}, 0xc036556d80, {0xc0109d5a20, 0x7}, ...)
	pkg/jobs/registry.go:1608 +0xba5 fp=0xc009b5fcd0 sp=0xc009b5f5b0 pc=0x2634565
github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).runJob(0xc025509508, {0x66707a0, 0xc002f1b400}, {0x666a0c0, 0xc00d00b040}, 0xc036556d80, {0xc0109d5a20, 0x7}, {0xc0311341f0, 0x8})
	pkg/jobs/adopt.go:414 +0x533 fp=0xc009b5fe98 sp=0xc009b5fcd0 pc=0x260fc93
github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).resumeJob.func1({0x66707a0?, 0xc002f1b0e0?})
	pkg/jobs/adopt.go:267 +0x13c fp=0xc009b5ff58 sp=0xc009b5fe98 pc=0x260e63c
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func1({0x66707a0, 0xc002f1b0e0})
	pkg/util/stop/stopper.go:438 +0x8a fp=0xc009b5ffc0 sp=0xc009b5ff58 pc=0x13d46ca
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.gowrap1()
	pkg/util/stop/stopper.go:439 +0x28 fp=0xc009b5ffe0 sp=0xc009b5ffc0 pc=0x13d4608
runtime.goexit({})
	src/runtime/asm_amd64.s:1700 +0x1 fp=0xc009b5ffe8 sp=0xc009b5ffe0 pc=0x4bb561
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx in goroutine 403354
	pkg/util/stop/stopper.go:436 +0x137
```
<details><summary>Log preceding fatal error</summary>
<p>

```
goos: linux
goarch: amd64
cpu: Intel(R) Xeon(R) CPU @ 2.60GHz
BenchmarkJobs
BenchmarkJobs/show_job
    test_log_scope.go:171: test logs captured to: /tmp/logBenchmarkJobs_show_job2681035150
    test_log_scope.go:82: use -show-logs to present logs inline
    testing_shims.go:40: -- test log scope end --
    test_log_scope.go:171: test logs captured to: /tmp/logBenchmarkJobs_show_job1046486689
    test_log_scope.go:82: use -show-logs to present logs inline
    testing_shims.go:40: -- test log scope end --
    test_log_scope.go:171: test logs captured to: /tmp/logBenchmarkJobs_show_job4286568118
    test_log_scope.go:82: use -show-logs to present logs inline

I250803 23:42:46.092574 1 (gostd) rand.go:268  [-] 1  random seed: -8292905210590692453
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*BenchmarkJobs.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-53194

pkg/bench/rttanalysis.BenchmarkJobs [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachprodMicrobenchmarkWeeklyWip/20322595?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachprodMicrobenchmarkWeeklyWip/20322595?buildTab=artifacts#./artifacts/microbench/experiment) on master @ [166d6538fc5bc47b861d81c95e9dbfc19994a1da](https://github.com/cockroachdb/cockroach/commits/166d6538fc5bc47b861d81c95e9dbfc19994a1da):

Fatal error:

```
panic: unexpectedly no result routers in {0xc0044008c0 [] [] {[]} 0 local}

```
Stack: 

```
goroutine 42759 gp=0xc0025863c0 m=34 mp=0xc003030008 [running]:
panic({0x55365a0?, 0xc02f59e870?})
	GOROOT/src/runtime/panic.go:811 +0x168 fp=0xc02a41c320 sp=0xc02a41c270 pc=0x4b24c8
github.com/cockroachdb/cockroach/pkg/sql/physicalplan.(*PhysicalPlan).GetResultTypes(0x4b093f?)
	pkg/sql/physicalplan/physical_plan.go:221 +0xb3 fp=0xc02a41c370 sp=0xc02a41c320 pc=0x297a8f3
github.com/cockroachdb/cockroach/pkg/sql/physicalplan.(*PhysicalPlan).EnsureSingleStreamOnGateway(0xc02fa60680, {0x66c9390, 0xc02fa22fa0}, 0x0)
	pkg/sql/physicalplan/physical_plan.go:502 +0x78 fp=0xc02a41c558 sp=0xc02a41c370 pc=0x297c3d8
github.com/cockroachdb/cockroach/pkg/sql.finalizePlanWithRowCount({0x66c9390, 0xc02fa22fa0}, 0x0?, 0xc02fa60680, 0x0?)
	pkg/sql/distsql_physical_planner.go:5575 +0x66 fp=0xc02a41c7e0 sp=0xc02a41c558 pc=0x3ee31a6
github.com/cockroachdb/cockroach/pkg/sql.FinalizePlan(...)
	pkg/sql/distsql_physical_planner.go:5562
github.com/cockroachdb/cockroach/pkg/sql/importer.distImport.func1(...)
	pkg/sql/importer/import_processor_planning.go:114
github.com/cockroachdb/cockroach/pkg/sql/importer.distImport({_, _}, {_, _}, _, _, {_, _, _}, {0x0, ...}, ...)
	pkg/sql/importer/import_processor_planning.go:118 +0x1719 fp=0xc02a41cbb0 sp=0xc02a41c7e0 pc=0x47b56b9
github.com/cockroachdb/cockroach/pkg/sql/importer.ingestWithRetry({_, _}, {_, _}, _, _, {_, _, _}, {0x0, ...}, ...)
	pkg/sql/importer/import_job.go:675 +0xa2f fp=0xc02a41cfc8 sp=0xc02a41cbb0 pc=0x47a656f
github.com/cockroachdb/cockroach/pkg/sql/importer.(*importResumer).Resume(0xc02a75c910, {0x66c7bb8, 0xc02a7573b0}, {0x5740820?, 0xc0312f3d98})
	pkg/sql/importer/import_job.go:238 +0x9d5 fp=0xc02a41d550 sp=0xc02a41cfc8 pc=0x47a2a75
github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine.func2(0xc0050d1980?, 0x0?, 0xc02a41d928, {0x66c2d20?, 0xc02a75c910?}, {0x66c7bb8?, 0xc02a7573b0?}, {0x5740820?, 0xc0312f3d98?})
	pkg/jobs/registry.go:1607 +0xb2 fp=0xc02a41d5b0 sp=0xc02a41d550 pc=0x26407f2
github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine(0xc00574ce08, {0x66c9390, 0xc02a75af00}, {0x5740820, 0xc0312f3d98}, {0x66c2d20, 0xc02a75c910}, 0xc0050d1980, {0xc03095df10, 0x7}, ...)
	pkg/jobs/registry.go:1608 +0xba5 fp=0xc02a41dcd0 sp=0xc02a41d5b0 pc=0x263ec85
github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).runJob(0xc00574ce08, {0x66c9390, 0xc02a75af00}, {0x66c2d20, 0xc02a75c910}, 0xc0050d1980, {0xc03095df10, 0x7}, {0xc030ae6690, 0x8})
	pkg/jobs/adopt.go:414 +0x533 fp=0xc02a41de98 sp=0xc02a41dcd0 pc=0x261a3b3
github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).resumeJob.func1({0x66c9390?, 0xc02a75ad20?})
	pkg/jobs/adopt.go:267 +0x13c fp=0xc02a41df58 sp=0xc02a41de98 pc=0x2618d5c
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func1({0x66c9390, 0xc02a75ad20})
	pkg/util/stop/stopper.go:438 +0x8a fp=0xc02a41dfc0 sp=0xc02a41df58 pc=0x13d9a6a
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.gowrap1()
	pkg/util/stop/stopper.go:439 +0x28 fp=0xc02a41dfe0 sp=0xc02a41dfc0 pc=0x13d99a8
runtime.goexit({})
	src/runtime/asm_amd64.s:1700 +0x1 fp=0xc02a41dfe8 sp=0xc02a41dfe0 pc=0x4bb561
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx in goroutine 42752
	pkg/util/stop/stopper.go:436 +0x137
```
<details><summary>Log preceding fatal error</summary>
<p>

```
goos: linux
goarch: amd64
cpu: Intel(R) Xeon(R) CPU @ 2.60GHz
BenchmarkJobs
BenchmarkJobs/show_job
    test_log_scope.go:171: test logs captured to: /tmp/logBenchmarkJobs_show_job3977149774
    test_log_scope.go:82: use -show-logs to present logs inline
    testing_shims.go:40: -- test log scope end --
    test_log_scope.go:171: test logs captured to: /tmp/logBenchmarkJobs_show_job2669116459
    test_log_scope.go:82: use -show-logs to present logs inline

I250810 11:21:56.422103 1 (gostd) rand.go:268  [-] 1  random seed: -3517823494483004750
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*BenchmarkJobs.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/bench/rttanalysis.BenchmarkJobs [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachprodMicrobenchmarkWeeklyWip/20353368?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachprodMicrobenchmarkWeeklyWip/20353368?buildTab=artifacts#./artifacts/microbench/experiment) on master @ [8f5ae52a8f9a419efd0c4ab26464ff9b265ad1f3](https://github.com/cockroachdb/cockroach/commits/8f5ae52a8f9a419efd0c4ab26464ff9b265ad1f3):

Fatal error:

```
panic: unexpectedly no result routers in {0xc02172b5e0 [] [] {[]} 0 local}

```
Stack: 

```
goroutine 504424 gp=0xc0066301e0 m=16 mp=0xc001a08008 [running]:
panic({0x553a9c0?, 0xc031ae6480?})
	GOROOT/src/runtime/panic.go:811 +0x168 fp=0xc01a26a320 sp=0xc01a26a270 pc=0x4b24c8
github.com/cockroachdb/cockroach/pkg/sql/physicalplan.(*PhysicalPlan).GetResultTypes(0x4b093f?)
	pkg/sql/physicalplan/physical_plan.go:221 +0xb3 fp=0xc01a26a370 sp=0xc01a26a320 pc=0x297b5b3
github.com/cockroachdb/cockroach/pkg/sql/physicalplan.(*PhysicalPlan).EnsureSingleStreamOnGateway(0xc015847300, {0x66d16b0, 0xc029c88960}, 0x0)
	pkg/sql/physicalplan/physical_plan.go:502 +0x78 fp=0xc01a26a558 sp=0xc01a26a370 pc=0x297d098
github.com/cockroachdb/cockroach/pkg/sql.finalizePlanWithRowCount({0x66d16b0, 0xc029c88960}, 0x0?, 0xc015847300, 0x0?)
	pkg/sql/distsql_physical_planner.go:5575 +0x66 fp=0xc01a26a7e0 sp=0xc01a26a558 pc=0x3ee45a6
github.com/cockroachdb/cockroach/pkg/sql.FinalizePlan(...)
	pkg/sql/distsql_physical_planner.go:5562
github.com/cockroachdb/cockroach/pkg/sql/importer.distImport.func1(...)
	pkg/sql/importer/import_processor_planning.go:114
github.com/cockroachdb/cockroach/pkg/sql/importer.distImport({_, _}, {_, _}, _, _, {_, _, _}, {0x0, ...}, ...)
	pkg/sql/importer/import_processor_planning.go:118 +0x1719 fp=0xc01a26abb0 sp=0xc01a26a7e0 pc=0x47b88d9
github.com/cockroachdb/cockroach/pkg/sql/importer.ingestWithRetry({_, _}, {_, _}, _, _, {_, _, _}, {0x0, ...}, ...)
	pkg/sql/importer/import_job.go:675 +0xa2f fp=0xc01a26afc8 sp=0xc01a26abb0 pc=0x47a978f
github.com/cockroachdb/cockroach/pkg/sql/importer.(*importResumer).Resume(0xc0243436d0, {0x66cfed8, 0xc020b65e90}, {0x57451c0?, 0xc001b3ef88})
	pkg/sql/importer/import_job.go:238 +0x9d5 fp=0xc01a26b550 sp=0xc01a26afc8 pc=0x47a5c95
github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine.func2(0xc03bc19680?, 0x0?, 0xc01a26b928, {0x66cb040?, 0xc0243436d0?}, {0x66cfed8?, 0xc020b65e90?}, {0x57451c0?, 0xc001b3ef88?})
	pkg/jobs/registry.go:1607 +0xb2 fp=0xc01a26b5b0 sp=0xc01a26b550 pc=0x2640e12
github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine(0xc005492e08, {0x66d16b0, 0xc03eac9220}, {0x57451c0, 0xc001b3ef88}, {0x66cb040, 0xc0243436d0}, 0xc03bc19680, {0xc019621620, 0x7}, ...)
	pkg/jobs/registry.go:1608 +0xba5 fp=0xc01a26bcd0 sp=0xc01a26b5b0 pc=0x263f2a5
github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).runJob(0xc005492e08, {0x66d16b0, 0xc03eac9220}, {0x66cb040, 0xc0243436d0}, 0xc03bc19680, {0xc019621620, 0x7}, {0xc019621bd0, 0x8})
	pkg/jobs/adopt.go:414 +0x533 fp=0xc01a26be98 sp=0xc01a26bcd0 pc=0x261a9d3
github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).resumeJob.func1({0x66d16b0?, 0xc03eac8e60?})
	pkg/jobs/adopt.go:267 +0x13c fp=0xc01a26bf58 sp=0xc01a26be98 pc=0x261937c
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func1({0x66d16b0, 0xc03eac8e60})
	pkg/util/stop/stopper.go:438 +0x8a fp=0xc01a26bfc0 sp=0xc01a26bf58 pc=0x13d508a
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.gowrap1()
	pkg/util/stop/stopper.go:439 +0x28 fp=0xc01a26bfe0 sp=0xc01a26bfc0 pc=0x13d4fc8
runtime.goexit({})
	src/runtime/asm_amd64.s:1700 +0x1 fp=0xc01a26bfe8 sp=0xc01a26bfe0 pc=0x4bb561
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx in goroutine 504420
	pkg/util/stop/stopper.go:436 +0x137
```
<details><summary>Log preceding fatal error</summary>
<p>

```
goos: linux
goarch: amd64
cpu: Intel(R) Xeon(R) CPU @ 2.60GHz
BenchmarkJobs
BenchmarkJobs/show_job
    test_log_scope.go:171: test logs captured to: /tmp/logBenchmarkJobs_show_job3457923042
    test_log_scope.go:82: use -show-logs to present logs inline
    testing_shims.go:40: -- test log scope end --
    test_log_scope.go:171: test logs captured to: /tmp/logBenchmarkJobs_show_job2117234573
    test_log_scope.go:82: use -show-logs to present logs inline
    testing_shims.go:40: -- test log scope end --
    test_log_scope.go:171: test logs captured to: /tmp/logBenchmarkJobs_show_job1185846947
    test_log_scope.go:82: use -show-logs to present logs inline

I250817 02:17:47.970918 1 (gostd) rand.go:268  [-] 1  random seed: 5643831601451359163
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*BenchmarkJobs.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/bench/rttanalysis.BenchmarkJobs [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachprodMicrobenchmarkWeeklyWip/20497571?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachprodMicrobenchmarkWeeklyWip/20497571?buildTab=artifacts#./artifacts/microbench/experiment) on master @ [223617d5f65835071f1f3395bc7b487e6dc029c9](https://github.com/cockroachdb/cockroach/commits/223617d5f65835071f1f3395bc7b487e6dc029c9):

Fatal error:

```
panic: unexpectedly no result routers in {0xc00aa3b5e0 [] [] {[]} 0 local}

```
Stack: 

```
goroutine 481360 gp=0xc036d1f6c0 m=25 mp=0xc00207a008 [running]:
panic({0x558fae0?, 0xc0416869e0?})
	GOROOT/src/runtime/panic.go:810 +0x168 fp=0xc007dba1b0 sp=0xc007dba100 pc=0x4ae548
github.com/cockroachdb/cockroach/pkg/sql/physicalplan.(*PhysicalPlan).GetResultTypes(0x140?)
	pkg/sql/physicalplan/physical_plan.go:221 +0xb3 fp=0xc007dba200 sp=0xc007dba1b0 pc=0x2a19873
github.com/cockroachdb/cockroach/pkg/sql/physicalplan.(*PhysicalPlan).EnsureSingleStreamOnGateway(0xc0144a7200, {0x6768df0, 0xc00400ce60}, 0x0)
	pkg/sql/physicalplan/physical_plan.go:502 +0x78 fp=0xc007dba3e8 sp=0xc007dba200 pc=0x2a1b358
github.com/cockroachdb/cockroach/pkg/sql.finalizePlanWithRowCount({0x6768df0, 0xc00400ce60}, 0x0?, 0xc0144a7200, 0x0?)
	pkg/sql/distsql_physical_planner.go:5585 +0x66 fp=0xc007dba670 sp=0xc007dba3e8 pc=0x4024686
github.com/cockroachdb/cockroach/pkg/sql.FinalizePlan(...)
	pkg/sql/distsql_physical_planner.go:5572
github.com/cockroachdb/cockroach/pkg/sql/importer.distImport.func1({0x6768df0, 0xc00400ce60}, 0xc0058466e0)
	pkg/sql/importer/import_processor_planning.go:118 +0x619 fp=0xc007dba938 sp=0xc007dba670 pc=0x48fbdf9
github.com/cockroachdb/cockroach/pkg/sql/importer.distImport({_, _}, {_, _}, _, _, {_, _, _}, {0x0, ...}, ...)
	pkg/sql/importer/import_processor_planning.go:122 +0x2b8 fp=0xc007dbab68 sp=0xc007dba938 pc=0x48f87f8
github.com/cockroachdb/cockroach/pkg/sql/importer.ingestWithRetry({_, _}, {_, _}, _, _, {_, _, _}, {0x0, ...}, ...)
	pkg/sql/importer/import_job.go:716 +0xa31 fp=0xc007dbaf80 sp=0xc007dbab68 pc=0x48eab71
github.com/cockroachdb/cockroach/pkg/sql/importer.(*importResumer).Resume(0xc01b8a5720, {0x6767598, 0xc024e2d530}, {0x57e2de0?, 0xc01ed2cdd0})
	pkg/sql/importer/import_job.go:250 +0xa16 fp=0xc007dbb550 sp=0xc007dbaf80 pc=0x48e6e16
github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine.func2(0xc005b52a80?, 0x0?, 0xc007dbb928, {0x67624a0?, 0xc01b8a5720?}, {0x6767598?, 0xc024e2d530?}, {0x57e2de0?, 0xc01ed2cdd0?})
	pkg/jobs/registry.go:1607 +0xb2 fp=0xc007dbb5b0 sp=0xc007dbb550 pc=0x26bd292
github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine(0xc026d90008, {0x6768df0, 0xc0140de8c0}, {0x57e2de0, 0xc01ed2cdd0}, {0x67624a0, 0xc01b8a5720}, 0xc005b52a80, {0xc0409c4bf0, 0x7}, ...)
	pkg/jobs/registry.go:1608 +0xba5 fp=0xc007dbbcd0 sp=0xc007dbb5b0 pc=0x26bb705
github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).runJob(0xc026d90008, {0x6768df0, 0xc0140de8c0}, {0x67624a0, 0xc01b8a5720}, 0xc005b52a80, {0xc0409c4bf0, 0x7}, {0xc01ba5e2d8, 0x8})
	pkg/jobs/adopt.go:414 +0x545 fp=0xc007dbbe98 sp=0xc007dbbcd0 pc=0x2696e85
github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).resumeJob.func1({0x6768df0?, 0xc026f3d7c0?})
	pkg/jobs/adopt.go:267 +0x13c fp=0xc007dbbf58 sp=0xc007dbbe98 pc=0x269581c
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func1({0x6768df0, 0xc026f3d7c0})
	pkg/util/stop/stopper.go:438 +0x8a fp=0xc007dbbfc0 sp=0xc007dbbf58 pc=0x1427a4a
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.gowrap1()
	pkg/util/stop/stopper.go:439 +0x28 fp=0xc007dbbfe0 sp=0xc007dbbfc0 pc=0x1427988
runtime.goexit({})
	src/runtime/asm_amd64.s:1700 +0x1 fp=0xc007dbbfe8 sp=0xc007dbbfe0 pc=0x4b7961
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx in goroutine 481356
	pkg/util/stop/stopper.go:436 +0x137
```
<details><summary>Log preceding fatal error</summary>
<p>

```
goos: linux
goarch: amd64
cpu: Intel(R) Xeon(R) CPU @ 2.60GHz
BenchmarkJobs
BenchmarkJobs/show_job
    test_log_scope.go:171: test logs captured to: /tmp/logBenchmarkJobs_show_job659942022
    test_log_scope.go:82: use -show-logs to present logs inline
    testing_shims.go:40: -- test log scope end --
    test_log_scope.go:171: test logs captured to: /tmp/logBenchmarkJobs_show_job4203967411
    test_log_scope.go:82: use -show-logs to present logs inline
    testing_shims.go:40: -- test log scope end --
    test_log_scope.go:171: test logs captured to: /tmp/logBenchmarkJobs_show_job618745550
    test_log_scope.go:82: use -show-logs to present logs inline

I250921 12:07:40.100221 1 (gostd) rand.go:268  [-] 1  random seed: -3075485161730235217
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*BenchmarkJobs.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/bench/rttanalysis.BenchmarkJobs [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachprodMicrobenchmarkWeeklyWip/20563166?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachprodMicrobenchmarkWeeklyWip/20563166?buildTab=artifacts#./artifacts/microbench/experiment) on master @ [5d15bbfbbbd4a7e8f8a96f183ad239ef48f5664f](https://github.com/cockroachdb/cockroach/commits/5d15bbfbbbd4a7e8f8a96f183ad239ef48f5664f):

Fatal error:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x2f8 pc=0x4929dba]

```
Stack: 

```
goroutine 52768 gp=0xc003755880 m=14 mp=0xc000881c08 [running]:
panic({0x528e620?, 0x9fb2e10?})
	GOROOT/src/runtime/panic.go:810 +0x168 fp=0xc01fcedbf0 sp=0xc01fcedb40 pc=0x4ae508
runtime.panicmem(...)
	GOROOT/src/runtime/panic.go:262
runtime.sigpanic()
	GOROOT/src/runtime/signal_unix.go:917 +0x359 fp=0xc01fcedc50 sp=0xc01fcedbf0 pc=0x4b0cd9
github.com/cockroachdb/cockroach/pkg/sql/importer.(*importResumer).prepareTableForIngestion(_, {_, _}, {_, _}, {{0x0, {0x0, 0x0}, 0x0, 0x0, ...}, ...}, ...)
	pkg/sql/importer/import_job.go:353 +0x15a fp=0xc01fcedff8 sp=0xc01fcedc50 pc=0x4929dba
github.com/cockroachdb/cockroach/pkg/sql/importer.(*importResumer).Resume.func1({0x67d4ef8, 0xc004fcc510}, {0x6850378, 0xc0200d1ae0}, 0xc004d4c000)
	pkg/sql/importer/import_job.go:144 +0x12b fp=0xc01fcee640 sp=0xc01fcedff8 pc=0x49296ab
github.com/cockroachdb/cockroach/pkg/sql.DescsTxn.func1({0x67d4ef8, 0xc004fcc510}, {0x6853768, 0xc0200d1ae0})
	pkg/sql/exec_util.go:4544 +0x76 fp=0xc01fcee688 sp=0xc01fcee640 pc=0x40c5d56
github.com/cockroachdb/cockroach/pkg/sql.DescsTxn.(*InternalDB).DescsTxn.func2({0x67d4ef8?, 0xc004fcc510?}, 0x68439c8?)
	pkg/sql/internal.go:1947 +0x2b fp=0xc01fcee6b8 sp=0xc01fcee688 pc=0x40c5cab
github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).txn.func4({0x67d4ef8, 0xc004fcc510}, 0xc0200cf4a0)
	pkg/sql/internal.go:2044 +0x354 fp=0xc01fcee968 sp=0xc01fcee6b8 pc=0x4103494
github.com/cockroachdb/cockroach/pkg/kv.(*Txn).exec(0xc0200cf4a0, {0x67d4ef8, 0xc004fcc510}, 0xc03c502840)
	pkg/kv/txn.go:1142 +0x1f1 fp=0xc01fceed58 sp=0xc01fcee968 pc=0x1ab9e91
github.com/cockroachdb/cockroach/pkg/kv.runTxn({0x67d4ef8, 0xc004fcc510}, 0xc0200cf4a0, 0x1?)
	pkg/kv/db.go:1092 +0x3e fp=0xc01fceedd8 sp=0xc01fceed58 pc=0x1ab08de
github.com/cockroachdb/cockroach/pkg/kv.(*DB).TxnWithAdmissionControl(0xc01fceee40?, {0x67d4ef8, 0xc004fcc510}, 0x58528a0?, 0x0?, 0x0, 0xc03c502840)
	pkg/kv/db.go:1055 +0xa5 fp=0xc01fceee18 sp=0xc01fceedd8 pc=0x1ab06a5
github.com/cockroachdb/cockroach/pkg/kv.(*DB).Txn(...)
	pkg/kv/db.go:1030
github.com/cockroachdb/cockroach/pkg/kv.(*DB).Txn-fm({0x67d4ef8?, 0xc004fcc510?}, 0x4a937d?)
	<autogenerated>:1 +0x3d fp=0xc01fceee50 sp=0xc01fceee18 pc=0x428309d
github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).txn(0xc005ba4f30, {0x67d4ef8, 0xc004fcc510}, 0xc0230c9440, {0x0, 0x0, 0x28?})
	pkg/sql/internal.go:2031 +0x28b fp=0xc01fceef30 sp=0xc01fceee50 pc=0x410306b
github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).DescsTxn(...)
	pkg/sql/internal.go:1945
github.com/cockroachdb/cockroach/pkg/sql.DescsTxn({0x67d4ef8, 0xc004fcc510}, 0xc01fceefe0?, 0xc014e53b30)
	pkg/sql/exec_util.go:4543 +0xbf fp=0xc01fceef88 sp=0xc01fceef30 pc=0x40c5c3f
github.com/cockroachdb/cockroach/pkg/sql/importer.(*importResumer).Resume(0xc00e1c1900, {0x67d4ef8, 0xc004fcc510}, {0x5843ea0?, 0xc002f176d8})
	pkg/sql/importer/import_job.go:141 +0x2e5 fp=0xc01fcef550 sp=0xc01fceef88 pc=0x4928405
github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine.func2(0xc0075c5c80?, 0x0?, 0xc01fcef928, {0x67cfde0?, 0xc00e1c1900?}, {0x67d4ef8?, 0xc004fcc510?}, {0x5843ea0?, 0xc002f176d8?})
	pkg/jobs/registry.go:1607 +0xb2 fp=0xc01fcef5b0 sp=0xc01fcef550 pc=0x26dc772
github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine(0xc003e70008, {0x67d6750, 0xc034db26e0}, {0x5843ea0, 0xc002f176d8}, {0x67cfde0, 0xc00e1c1900}, 0xc0075c5c80, {0xc00d6cbce0, 0x7}, ...)
	pkg/jobs/registry.go:1608 +0xba5 fp=0xc01fcefcd0 sp=0xc01fcef5b0 pc=0x26dabe5
github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).runJob(0xc003e70008, {0x67d6750, 0xc034db26e0}, {0x67cfde0, 0xc00e1c1900}, 0xc0075c5c80, {0xc00d6cbce0, 0x7}, {0xc00b7fa320, 0x8})
	pkg/jobs/adopt.go:414 +0x545 fp=0xc01fcefe98 sp=0xc01fcefcd0 pc=0x26b5e45
github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).resumeJob.func1({0x67d6750?, 0xc034db2500?})
	pkg/jobs/adopt.go:267 +0x13c fp=0xc01fceff58 sp=0xc01fcefe98 pc=0x26b47dc
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func1({0x67d6750, 0xc034db2500})
	pkg/util/stop/stopper.go:440 +0x8a fp=0xc01fceffc0 sp=0xc01fceff58 pc=0x143052a
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.gowrap1()
	pkg/util/stop/stopper.go:441 +0x28 fp=0xc01fceffe0 sp=0xc01fceffc0 pc=0x1430468
runtime.goexit({})
	src/runtime/asm_amd64.s:1700 +0x1 fp=0xc01fceffe8 sp=0xc01fceffe0 pc=0x4b7921
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx in goroutine 53189
	pkg/util/stop/stopper.go:438 +0x137
```
<details><summary>Log preceding fatal error</summary>
<p>

```
goos: linux
goarch: amd64
cpu: Intel(R) Xeon(R) CPU @ 2.60GHz
BenchmarkJobs
BenchmarkJobs/show_job
    test_log_scope.go:171: test logs captured to: /tmp/logBenchmarkJobs_show_job3147900160
    test_log_scope.go:82: use -show-logs to present logs inline
    testing_shims.go:40: -- test log scope end --
    test_log_scope.go:171: test logs captured to: /tmp/logBenchmarkJobs_show_job1439110429
    test_log_scope.go:82: use -show-logs to present logs inline

I251005 17:27:40.259089 1 (gostd) rand.go:268  [-] 1  random seed: -377103227595789302
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*BenchmarkJobs.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/bench/rttanalysis.BenchmarkJobs [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachprodMicrobenchmarkWeeklyWip/20652913?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachprodMicrobenchmarkWeeklyWip/20652913?buildTab=artifacts#./artifacts/microbench/experiment) on master @ [cdcf76d0af53d78418ed10585fca7987c1c0aedc](https://github.com/cockroachdb/cockroach/commits/cdcf76d0af53d78418ed10585fca7987c1c0aedc):

Fatal error:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x2f8 pc=0x496347a]

```
Stack: 

```
goroutine 49586 gp=0xc00130e380 m=16 mp=0xc001a00008 [running]:
panic({0x52d5f80?, 0xa046e20?})
	GOROOT/src/runtime/panic.go:810 +0x168 fp=0xc02d0b3b50 sp=0xc02d0b3aa0 pc=0x4ae508
runtime.panicmem(...)
	GOROOT/src/runtime/panic.go:262
runtime.sigpanic()
	GOROOT/src/runtime/signal_unix.go:917 +0x359 fp=0xc02d0b3bb0 sp=0xc02d0b3b50 pc=0x4b0cd9
github.com/cockroachdb/cockroach/pkg/sql/importer.(*importResumer).prepareTableForIngestion(_, {_, _}, {_, _}, {{0x0, {0x0, 0x0}, 0x0, 0x0, ...}, ...}, ...)
	pkg/sql/importer/import_job.go:400 +0x15a fp=0xc02d0b3f58 sp=0xc02d0b3bb0 pc=0x496347a
github.com/cockroachdb/cockroach/pkg/sql/importer.(*importResumer).Resume.func1({0x6830b78, 0xc02cd530e0}, {0x68acc38, 0xc02d620aa0}, 0xc02d3d3080)
	pkg/sql/importer/import_job.go:173 +0x12b fp=0xc02d0b45a0 sp=0xc02d0b3f58 pc=0x4962d6b
github.com/cockroachdb/cockroach/pkg/sql.DescsTxn.func1({0x6830b78, 0xc02cd530e0}, {0x68afee8, 0xc02d620aa0})
	pkg/sql/exec_util.go:4561 +0x76 fp=0xc02d0b45e8 sp=0xc02d0b45a0 pc=0x40f8df6
github.com/cockroachdb/cockroach/pkg/sql.DescsTxn.(*InternalDB).DescsTxn.func2({0x6830b78?, 0xc02cd530e0?}, 0x68a00f0?)
	pkg/sql/internal.go:1960 +0x2b fp=0xc02d0b4618 sp=0xc02d0b45e8 pc=0x40f8d4b
github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).txn.func4({0x6830b78, 0xc02cd530e0}, 0xc02d3f4b00)
	pkg/sql/internal.go:2057 +0x354 fp=0xc02d0b48c8 sp=0xc02d0b4618 pc=0x4137a94
github.com/cockroachdb/cockroach/pkg/kv.(*Txn).exec(0xc02d3f4b00, {0x6830b78, 0xc02cd530e0}, 0xc025e03b40)
	pkg/kv/txn.go:1144 +0x1f1 fp=0xc02d0b4cb8 sp=0xc02d0b48c8 pc=0x1adb3d1
github.com/cockroachdb/cockroach/pkg/kv.runTxn({0x6830b78, 0xc02cd530e0}, 0xc02d3f4b00, 0x1?)
	pkg/kv/db.go:1092 +0x3e fp=0xc02d0b4d38 sp=0xc02d0b4cb8 pc=0x1ad1dfe
github.com/cockroachdb/cockroach/pkg/kv.(*DB).TxnWithAdmissionControl(0xc02d0b4da0?, {0x6830b78, 0xc02cd530e0}, 0x58a0c00?, 0x0?, 0x0, 0xc025e03b40)
	pkg/kv/db.go:1055 +0xa5 fp=0xc02d0b4d78 sp=0xc02d0b4d38 pc=0x1ad1bc5
github.com/cockroachdb/cockroach/pkg/kv.(*DB).Txn(...)
	pkg/kv/db.go:1030
github.com/cockroachdb/cockroach/pkg/kv.(*DB).Txn-fm({0x6830b78?, 0xc02cd530e0?}, 0x4a937d?)
	<autogenerated>:1 +0x3d fp=0xc02d0b4db0 sp=0xc02d0b4d78 pc=0x42b83bd
github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).txn(0xc0056777a0, {0x6830b78, 0xc02cd530e0}, 0xc02d3ce5c0, {0x0, 0x0, 0x28?})
	pkg/sql/internal.go:2044 +0x28b fp=0xc02d0b4e90 sp=0xc02d0b4db0 pc=0x413766b
github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).DescsTxn(...)
	pkg/sql/internal.go:1958
github.com/cockroachdb/cockroach/pkg/sql.DescsTxn({0x6830b78, 0xc02cd530e0}, 0x42b8380?, 0xc02d3dbec0)
	pkg/sql/exec_util.go:4560 +0xbf fp=0xc02d0b4ee8 sp=0xc02d0b4e90 pc=0x40f8cdf
github.com/cockroachdb/cockroach/pkg/sql/importer.(*importResumer).Resume(0xc02d08e050, {0x6830b78, 0xc02cd530e0}, {0x5891e00?, 0xc00285d2b8})
	pkg/sql/importer/import_job.go:170 +0x2e5 fp=0xc02d0b5550 sp=0xc02d0b4ee8 pc=0x4961765
github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine.func2(0xc0129c8900?, 0x0?, 0xc02d0b5928, {0x682b970?, 0xc02d08e050?}, {0x6830b78?, 0xc02cd530e0?}, {0x5891e00?, 0xc00285d2b8?})
	pkg/jobs/registry.go:1641 +0xb2 fp=0xc02d0b55b0 sp=0xc02d0b5550 pc=0x27093f2
github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine(0xc002864808, {0x6832388, 0xc02c595860}, {0x5891e00, 0xc00285d2b8}, {0x682b970, 0xc02d08e050}, 0xc0129c8900, {0xc005ec9c90, 0x7}, ...)
	pkg/jobs/registry.go:1642 +0xba5 fp=0xc02d0b5cd0 sp=0xc02d0b55b0 pc=0x2707865
github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).runJob(0xc002864808, {0x6832388, 0xc02c595860}, {0x682b970, 0xc02d08e050}, 0xc0129c8900, {0xc005ec9c90, 0x7}, {0xc00a1598b0, 0x8})
	pkg/jobs/adopt.go:415 +0x545 fp=0xc02d0b5e98 sp=0xc02d0b5cd0 pc=0x26e2525
github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).resumeJob.func1({0x6832388?, 0xc02c595680?})
	pkg/jobs/adopt.go:268 +0x13c fp=0xc02d0b5f58 sp=0xc02d0b5e98 pc=0x26e0ebc
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func1({0x6832388, 0xc02c595680})
	pkg/util/stop/stopper.go:440 +0x8a fp=0xc02d0b5fc0 sp=0xc02d0b5f58 pc=0x144724a
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.gowrap1()
	pkg/util/stop/stopper.go:441 +0x28 fp=0xc02d0b5fe0 sp=0xc02d0b5fc0 pc=0x1447188
runtime.goexit({})
	src/runtime/asm_amd64.s:1700 +0x1 fp=0xc02d0b5fe8 sp=0xc02d0b5fe0 pc=0x4b7921
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx in goroutine 49556
	pkg/util/stop/stopper.go:438 +0x137
```
<details><summary>Log preceding fatal error</summary>
<p>

```
goos: linux
goarch: amd64
cpu: Intel(R) Xeon(R) CPU @ 2.60GHz
BenchmarkJobs
BenchmarkJobs/show_job
    test_log_scope.go:171: test logs captured to: /tmp/logBenchmarkJobs_show_job3800666128
    test_log_scope.go:82: use -show-logs to present logs inline
    testing_shims.go:40: -- test log scope end --
    test_log_scope.go:171: test logs captured to: /tmp/logBenchmarkJobs_show_job4006748410
    test_log_scope.go:82: use -show-logs to present logs inline

I251026 13:57:27.346094 1 (gostd) rand.go:268  [-] 1  random seed: -2198714454600906139
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*BenchmarkJobs.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/bench/rttanalysis.BenchmarkJobs [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachprodMicrobenchmarkWeeklyWip/20685401?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachprodMicrobenchmarkWeeklyWip/20685401?buildTab=artifacts#./artifacts/microbench/experiment) on master @ [cc61b3c16062dfeef5ee82e6a80d868d499e047e](https://github.com/cockroachdb/cockroach/commits/cc61b3c16062dfeef5ee82e6a80d868d499e047e):

Fatal error:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x2f8 pc=0x4932bda]

```
Stack: 

```
goroutine 43001 gp=0xc003eb10e0 m=24 mp=0xc000900808 [running]:
panic({0x5316180?, 0xa278e70?})
	GOROOT/src/runtime/panic.go:802 +0x168 fp=0xc02201da18 sp=0xc02201d968 pc=0x4ba208
runtime.panicmem(...)
	GOROOT/src/runtime/panic.go:262
runtime.sigpanic()
	GOROOT/src/runtime/signal_unix.go:925 +0x359 fp=0xc02201da78 sp=0xc02201da18 pc=0x4bcdd9
github.com/cockroachdb/cockroach/pkg/sql/importer.(*importResumer).prepareTableForIngestion(_, {_, _}, {_, _}, {{0x0, {0x0, 0x0}, 0x0, 0x0, ...}, ...}, ...)
	pkg/sql/importer/import_job.go:401 +0x15a fp=0xc02201de20 sp=0xc02201da78 pc=0x4932bda
github.com/cockroachdb/cockroach/pkg/sql/importer.(*importResumer).Resume.func1({0x6963618, 0xc0145df350}, {0x69e04f8, 0xc02298b5e0}, 0xc01244de00)
	pkg/sql/importer/import_job.go:174 +0x12b fp=0xc02201e468 sp=0xc02201de20 pc=0x49324cb
github.com/cockroachdb/cockroach/pkg/sql.DescsTxn.func1({0x6963618, 0xc0145df350}, {0x69e38e8, 0xc02298b5e0})
	pkg/sql/exec_util.go:3494 +0x76 fp=0xc02201e4b0 sp=0xc02201e468 pc=0x4095356
github.com/cockroachdb/cockroach/pkg/sql.DescsTxn.(*InternalDB).DescsTxn.func2({0x6963618?, 0xc0145df350?}, 0x69d3b90?)
	pkg/sql/internal.go:1961 +0x2b fp=0xc02201e4e0 sp=0xc02201e4b0 pc=0x40952ab
github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).txn.func4({0x6963618, 0xc0145df350}, 0xc02265f600)
	pkg/sql/internal.go:2058 +0x33e fp=0xc02201e790 sp=0xc02201e4e0 pc=0x40d353e
github.com/cockroachdb/cockroach/pkg/kv.(*Txn).exec(0xc02265f600, {0x6963618, 0xc0145df350}, 0xc022109b80)
	pkg/kv/txn.go:1144 +0x1f1 fp=0xc02201eb80 sp=0xc02201e790 pc=0x1abf3d1
github.com/cockroachdb/cockroach/pkg/kv.runTxn({0x6963618, 0xc0145df350}, 0xc02265f600, 0x1?)
	pkg/kv/db.go:1092 +0x3e fp=0xc02201ec00 sp=0xc02201eb80 pc=0x1ab5fbe
github.com/cockroachdb/cockroach/pkg/kv.(*DB).TxnWithAdmissionControl(0xc02201ec68?, {0x6963618, 0xc0145df350}, 0x5943a80?, 0x0?, 0x0, 0xc022109b80)
	pkg/kv/db.go:1055 +0xa5 fp=0xc02201ec40 sp=0xc02201ec00 pc=0x1ab5d85
github.com/cockroachdb/cockroach/pkg/kv.(*DB).Txn(...)
	pkg/kv/db.go:1030
github.com/cockroachdb/cockroach/pkg/kv.(*DB).Txn-fm({0x6963618?, 0xc0145df350?}, 0xc02201ecd0?)
	<autogenerated>:1 +0x3d fp=0xc02201ec78 sp=0xc02201ec40 pc=0x4244afd
github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).txn(0xc006492120, {0x6963618, 0xc0145df350}, 0xc022d48040, {0x0, 0x0, 0x28?})
	pkg/sql/internal.go:2045 +0x27a fp=0xc02201ee78 sp=0xc02201ec78 pc=0x40d2bda
github.com/cockroachdb/cockroach/pkg/sql.(*InternalDB).DescsTxn(...)
	pkg/sql/internal.go:1959
github.com/cockroachdb/cockroach/pkg/sql.DescsTxn({0x6963618, 0xc0145df350}, 0x455fa5?, 0xc02299cf60)
	pkg/sql/exec_util.go:3493 +0xbf fp=0xc02201eed0 sp=0xc02201ee78 pc=0x409523f
github.com/cockroachdb/cockroach/pkg/sql/importer.(*importResumer).Resume(0xc0145a1c20, {0x6963618, 0xc0145df350}, {0x5932880?, 0xc00e050180})
	pkg/sql/importer/import_job.go:171 +0x2c5 fp=0xc02201f538 sp=0xc02201eed0 pc=0x4930f05
github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine.func2(0xc0145c4300?, 0x0?, 0xc02201f910, {0x695d0d0?, 0xc0145a1c20?}, {0x6963618?, 0xc0145df350?}, {0x5932880?, 0xc00e050180?})
	pkg/jobs/registry.go:1641 +0xb4 fp=0xc02201f598 sp=0xc02201f538 pc=0x26d2b14
github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).stepThroughStateMachine(0xc00323b808, {0x6964df0, 0xc022294000}, {0x5932880, 0xc00e050180}, {0x695d0d0, 0xc0145a1c20}, 0xc0145c4300, {0xc00ccfa600, 0x7}, ...)
	pkg/jobs/registry.go:1642 +0xb7f fp=0xc02201fcb8 sp=0xc02201f598 pc=0x26d0f9f
github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).runJob(0xc00323b808, {0x6964df0, 0xc022294000}, {0x695d0d0, 0xc0145a1c20}, 0xc0145c4300, {0xc00ccfa600, 0x7}, {0xc00ccfab20, 0x8})
	pkg/jobs/adopt.go:415 +0x4f5 fp=0xc02201fe98 sp=0xc02201fcb8 pc=0x26abb75
github.com/cockroachdb/cockroach/pkg/jobs.(*Registry).resumeJob.func1({0x6964df0?, 0xc0145c3d60?})
	pkg/jobs/adopt.go:268 +0x13c fp=0xc02201ff58 sp=0xc02201fe98 pc=0x26aa55c
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func1({0x6964df0, 0xc0145c3d60})
	pkg/util/stop/stopper.go:440 +0x8a fp=0xc02201ffc0 sp=0xc02201ff58 pc=0x1423a2a
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.gowrap1()
	pkg/util/stop/stopper.go:441 +0x28 fp=0xc02201ffe0 sp=0xc02201ffc0 pc=0x1423968
runtime.goexit({})
	src/runtime/asm_amd64.s:1693 +0x1 fp=0xc02201ffe8 sp=0xc02201ffe0 pc=0x4c36e1
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx in goroutine 43098
	pkg/util/stop/stopper.go:438 +0x125
```
<details><summary>Log preceding fatal error</summary>
<p>

```
goos: linux
goarch: amd64
cpu: Intel(R) Xeon(R) CPU @ 2.60GHz
BenchmarkJobs
BenchmarkJobs/show_job
    test_log_scope.go:171: test logs captured to: /tmp/logBenchmarkJobs_show_job1062346035
    test_log_scope.go:82: use -show-logs to present logs inline
    testing_shims.go:40: -- test log scope end --
    test_log_scope.go:171: test logs captured to: /tmp/logBenchmarkJobs_show_job664163484
    test_log_scope.go:82: use -show-logs to present logs inline

I251102 21:56:43.119332 1 (gostd) rand.go:268  [-] 1  random seed: -5381493981352963273
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*BenchmarkJobs.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/bench/rttanalysis/BUILD.bazel
+++ b/pkg/bench/rttanalysis/BUILD.bazel
@@ -65,6 +65,7 @@ go_test(
         "//pkg/security/username",
         "//pkg/server",
         "//pkg/server/serverpb",
+        "//pkg/sql/catalog/descpb",
         "//pkg/testutils/pgurlutils",
         "//pkg/testutils/serverutils",
         "//pkg/testutils/skip",

--- a/pkg/bench/rttanalysis/jobs_test.go
+++ b/pkg/bench/rttanalysis/jobs_test.go
@@ -14,13 +14,27 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/security/username"
 	"github.com/cockroachdb/cockroach/pkg/server"
 	"github.com/cockroachdb/cockroach/pkg/server/serverpb"
+	"github.com/cockroachdb/cockroach/pkg/sql/catalog/descpb"
 	"github.com/cockroachdb/cockroach/pkg/util/protoutil"
 )
 
 func BenchmarkJobs(b *testing.B) { reg.Run(b) }
 func init() {
+	// Create a minimal table descriptor for the import job.
+	tableDesc := &descpb.TableDescriptor{
+		ID:            100,
+		ParentID:      1,
+		Name:          "benchmark_table",
+		FormatVersion: descpb.InterleavedFormatVersion,
+		Version:       1,
+	}
+
 	payloadBytes, err := protoutil.Marshal(&jobspb.Payload{
-		Details:       jobspb.WrapPayloadDetails(jobspb.ImportDetails{}),
+		Details: jobspb.WrapPayloadDetails(jobspb.ImportDetails{
+			Table: jobspb.ImportDetails_Table{
+				Desc: tableDesc,
+			},
+		}),
 		UsernameProto: username.RootUserName().EncodeProto(),
 	})
 	if err != nil {

